src/app/components/header/header.component.html

```diff
@@ -0,0 +1,13 @@
+<header>
+    <nav class="left-nav">
+
+        <ul>
+            <li class="h6"><a [routerLink]="['/']">HomePage</a></li>
+            <li class="h6"><a [routerLink]="['/List']">Listing</a></li>
+        </ul>
+    </nav>
+    <nav class="right-nav">
+        <button type="button" class="btn btn-success">Login</button>
+        <a href=""><li><img src="" alt=""></li></a>
```

Comment:
why do you have an empty image within a li tag - within an empty hyperlink (a) tag?

Reply:
Removed that, at that time I was thinking of adding some custom icon beside login, later dropped the idea but left this there only.